core/dbt/adapters/base/impl.py

```diff
@@ -1059,7 +1063,7 @@ def calculate_freshness(
         table = self.execute_macro(
             FRESHNESS_MACRO_NAME,
             kwargs=kwargs,
-            release=True,
+            release=False,
```

Comment:
is the `release` arg used anymore and should it stick around if not?

Reply:
It's not used in core anymore, but it is used in some plugins. Maybe anything using it needs to be changed anyway, I just hate to cause more plugin churn.

Reply:
got it, sounds good

Reply:
I've played around with this some, and I think we have to remove the release arg.


@jtcohen6 I think this will break any dependent adapters. I am mostly fine with that happening in a patch release since it fixes a pretty crippling bug. Ours all are locked to minor versions so we'll be fine... let me know what you think!

Reply:
Ok! I'm on board. This is a tiny breaking change to resolve a much more significant regression. We'll want to check that our plugins (spark + presto) can work with `0.17.2-rc1`.